ts/src/vertex.ts

```diff
@@ -0,0 +1,2796 @@
+        //       "base_filled": 175,
+        //       "quote_filled": -198.90714214264278,
+        //       "timestamp": 1691068943,
+        //       "trade_type": "buy"
+        // }
+        // fetchMytrades
+        // {
+        //     "digest": "0x80ce789702b670b7d33f2aa67e12c85f124395c3f9acdb422dde3b4973ccd50c",
+        //     "order": {
+        //         "sender": "0x12a0b4888021576eb10a67616dd3dd3d9ce206b664656661756c740000000000",
+        //         "priceX18": "27544000000000000000000",
+        //         "amount": "2000000000000000000",
+        //         "expiration": "4611686020107119633",
+        //         "nonce": "1761322608857448448"
+        //     },
+        //     "base_filled": "736000000000000000",
+        //     "quote_filled": "-20276464287857571514302",
+        //     "fee": "4055287857571514302",
+        //     "sequencer_fee": "0"
+        //     "cumulative_fee": "4055287857571514302",
+        //     "cumulative_base_filled": "736000000000000000",
+        //     "cumulative_quote_filled": "-20276464287857571514302",
+        //     "submission_idx": "563012",
+        //     "pre_balance": {
+        //       "base": {
+        //         "perp": {
+        //           "product_id": 2,
+        //           "lp_balance": {
+        //             "amount": "0",
+        //             "last_cumulative_funding_x18": "1823351297710837"
+        //           },
+        //           "balance": {
+        //             "amount": "2686684000000000000000",
+        //             "v_quote_balance": "-76348662407149297671587247",
+        //             "last_cumulative_funding_x18": "134999841911604906604576"
+        //           }
+        //         }
+        //       },
+        //       "quote": null
+        //     },
+        //     "post_balance": {
+        //       "base": {
+        //         "perp": {
+        //           "product_id": 2,
+        //           "lp_balance": {
+        //             "amount": "0",
+        //             "last_cumulative_funding_x18": "1823351297710837"
+        //           },
+        //           "balance": {
+        //             "amount": "2686013000000000000000",
+        //             "v_quote_balance": "-76328351274188497671587247",
+        //             "last_cumulative_funding_x18": "134999841911604906604576"
+        //           }
+        //         }
+        //       },
+        //       "quote": null
+        //     }
+        //   }
+        let price = undefined;
+        let amount = undefined;
+        let side = undefined;
+        let fee = undefined;
+        const id = this.safeString2 (trade, 'trade_id', 'submission_idx');
+        const order = this.safeString (trade, 'digest');
+        const timestamp = this.safeTimestamp (trade, 'timestamp');
+        if (timestamp === undefined) {
+            // fetchMyTrades
+            const baseBalance = this.safeDict (this.safeDict (trade, 'pre_balance', {}), 'base', {});
```

Comment:
@sc0Vu fetchMyTrades is not returning amount/price/cost/etc
![image](https://github.com/ccxt/ccxt/assets/43336371/8b00961d-d1ab-423b-b212-fd5085d887a4)